Source/KantanDocGen/Private/NodeDocsGenerator.cpp

```diff
@@ -602,6 +602,17 @@ bool FNodeDocsGenerator::GenerateTypeMembers(UObject* Type)
 						}
 					}
 				}
+				else
+				{
+					// Avoid any property that is part of the superclass and then "redefined" in this Class
+					bool IsInSuper = PropertyIterator->IsInContainer(ClassInstance->GetSuperClass());
+					if (IsInSuper == false)
```

Comment:
I found that, for some types, the reflection system would say a doc was not generated for a super class property. This conditional makes sure that such property is defined within the ClassIntance